modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/util/comparator/OrganizationMemberNameComparator.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.users.admin.web.util.comparator;
+
+import com.liferay.portal.kernel.model.Organization;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.util.OrderByComparator;
+
+/**
+ * @author Pei-Jung Lan
+ */
+public class OrganizationMemberNameComparator
+	extends OrderByComparator<Object> {
+
+	public static final String ORDER_BY_ASC = "name ASC";
+
+	public static final String ORDER_BY_DESC = "name DESC";
+
+	public static final String[] ORDER_BY_FIELDS = {"name"};
+
+	public OrganizationMemberNameComparator() {
+		this(false);
+	}
+
+	public OrganizationMemberNameComparator(boolean ascending) {
+		_ascending = ascending;
+	}
+
+	@Override
+	public int compare(Object object1, Object object2) {
+		String name1;
+		String name2;
+
+		if (object1 instanceof Organization) {
```

Comment:
Hey Pei-Jung,

Regarding the sentence you don't understand, it's because GitHub was not showing it correctly. I've now escaped the greater than symbol so that it properly shows List\<Object> vs List\<RepositoryEntry>

Why do you think that introducing an interface will cause that? The interface will only be used in the new methods which are already causing a change in the minor version, don't they?

In any case, this is a suggestion, if you want to follow RepositoryModelTitleComparator, that's fine too.